src/app.ts

```diff
@@ -0,0 +1,28 @@
+import * as express from 'express'
+import { Application } from 'express'
+
+class App {
+    public app: Application;
+    public port: number;
+
+    constructor(appInit: { port: number; middleWares }) {
+        this.app = express();
+        this.port = appInit.port;
+
+        this.middlewares(appInit.middleWares);
+    }
+
+    private middlewares(middleWares: { forEach: (arg0: (middleWare) => void) => void }): void {
```

Comment:
I am not sure if `(middleWares: { forEach: (arg0: (middleWare) => void) => void }` is a proper way to declare an array type. It can be `function[]` or similar